README.md

```diff
@@ -8,22 +8,23 @@ Apache Drill is a distributed MPP query layer that supports SQL and alternative
 
 ## Developers
 
-Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md).
+Please read [Environment.md](docs/dev/Environment.md) for setting up and running Apache Drill. For complete developer documentation see [DevDocs.md](docs/dev/DevDocs.md) and 
+also use slack channel for discussions.
```

Comment:
I'll suggest skipping this change

Reply:
@Agirish What's the objection to publicizing the Slack channel?